### PR TITLE
Remove load_deploytool

### PR DESCRIPTION
**Pull request by mkolesnik**
Since SUBM_NS is now declared in utils in shipyard, no need for this
copied code and this flag.

Also removed cleanup method which was left undeleted by mistake.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ CLUSTERS_ARGS = --cluster_settings $(DAPPER_SOURCE)/scripts/kind-e2e/cluster_set
 clusters: build images
 
 e2e: # internally depends on deploy target, which will execute only if not already deployed
-	./scripts/kind-e2e/e2e.sh --deploytool $(deploytool)
+	./scripts/kind-e2e/e2e.sh
```
Comment (mangelajo) [inline Makefile:16]: Don't we need to pass the flags down ? how's it working now that we are calling deploy from the kind e2e? :?

Comment (mangelajo) [reply Makefile:16]: Hmm, I see it was working for globalnet that we don't pass, how? :?

Comment (mkolesnik) [reply Makefile]: it's magic of make :)
it's passed since it's still declared in .travis.yaml

Comment (mangelajo) [reply Makefile]: But .. hmmm I see this:   - env: CMD="make e2e deploytool=helm globalnet=true"

How are those passed down anymore if we remove the flags when calling e2e.sh ?

Comment (mangelajo) [reply Makefile]: Looking at CI it's clearly working, so I'll keep the questions offline ;)

**Pull request merged**